README.md

```diff
@@ -57,7 +57,7 @@ def duplicates?(array)
 end
 ```
 
-Snippet 5 - Big O:
+Snippet 5 - Big O: O(1) = If I'm interpreting this correctly, these are effectively .times loops because the size of the arrays is defined as words.length = 3 and endings.length = 3, and two nested O(1) loops just gives an overall O(1) value. However, if the size of words or endings was a variable, this would end up being an O(n^2) efficiency, as described in Snippet 4 above.
```

Comment:
Ok ok... I lied a little. On accident. This would actually be O(n*m) because the two lists are different. But it's also the one that I mentioned in class that I always accidentally mix up with O(n^2). 

But your reasoning, on O(1) vs not is good.